iOS Gesture Alphabet/ViewController.swift

```diff
@@ -6,14 +6,53 @@
 //
 
 import UIKit
+import CoreMotion
 
 class ViewController: UIViewController {
 
+    
+    @IBOutlet weak var xAccel: UITextField!
+    @IBOutlet weak var yAccel: UITextField!
+    @IBOutlet weak var zAccel: UITextField!
+    
+    var motion = CMMotionManager()
+    
     override func viewDidLoad() {
```

Comment:
Did we want to do this format instead for functions:
`func name()
{
    func stuff
}`
Or should we keep it as the coding standard suggests where the '{' is on the same line as the function name?

Reply:
Either is fine with me. I should've been paying more attention, but that was what xCode just implemented on its own. We can and likely should stick with the coding standards

Reply:
Since xCode is auto implementing it the way that the coding standard showed, let's stick with that.